packages/api/src/graphql/mutation.ts

```diff
@@ -422,7 +422,6 @@ export const GraphQLAdminMutation = new GraphQLObjectType<undefined, Context>({
         const {roles} = authenticate()
         authorise(CanDeleteNavigation, roles)
         await dbAdapter.navigation.deleteNavigation(id)
-        return dbAdapter.navigation.deleteNavigation(id)
```

Comment:
and now you return nothing???  Did you test that?

Reply:
Yes I did test it and it does work:
![Peek 2020-11-18 13-06](https://user-images.githubusercontent.com/11842633/99531488-14c82080-29a3-11eb-81cd-b606e6cb87da.gif)

Actually, if I use the same code-pattern as in `deleteAuthor` it would throw a typescript-error

![Screenshot-20201118133906-795x218](https://user-images.githubusercontent.com/11842633/99531730-76888a80-29a3-11eb-9560-a026aa155992.png)

and also not work anymore when running:

![Screenshot-20201118133329-375x199](https://user-images.githubusercontent.com/11842633/99531909-bd768000-29a3-11eb-9b0f-8eedc090be84.png)

I don't really understand why it behaves in that way, I assume there must be somewhere a bug left from duda.